library/general/test/arch_test.rb

```diff
@@ -0,0 +1,47 @@
+#! /usr/bin/env rspec
+
+require_relative "test_helper"
+
+Yast.import "Arch"
+
+require "yast"
+
+describe Yast::Arch do
+
+  describe "DetectzKVM" do
```

Comment:
describe should match method, so 
```
describe ".is_zkvm" do
```
so if you run rspec --format doc it will write class, its methods and then its specification.